src/__init__.py

```diff
@@ -1,3 +1,9 @@
 from config import Config
+from .database.db_connection import DBConnection
 
-run_config = Config()
\ No newline at end of file
+run_config = Config()
+try:
+    db = DBConnection(run_config.db_file)
+    db.create_session()
+except Exception as e:
+    run_config.log_event.info(e)
```

Comment:
Should there be more information logged here other than just the exception string?